helix-view/src/document.rs

```diff
@@ -96,6 +96,9 @@ pub struct Document {
     /// Current indent style.
     pub indent_style: IndentStyle,
 
+    /// An override of the language-specified tab width.
+    pub tab_width_override: Option<usize>,
```

Comment:
I think this should be private: we probably want to be using the `tab_width` method instead of having direct access to this field. What do you think?

Reply:
I thought about that as well. My instinct was to make it public to match the rest of the indentation-related members as well as make it easier to change. But you're right, it's probably better to keep it private and add a method to change it, just to prevent any code from reading it when it shouldn't.